easybuild/easyconfigs/s/sep/sep-1.0.3-intel-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'sep'
+version = '1.0.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://sep.readthedocs.io/en/v1.0.x/'
+homepage = 'https://sep.readthedocs.io/en/v1.0.x/'
```

Comment:
@jose-d sorry to keep picking but the homepage line is now duplicated

also, simply `https://sep.readthedocs.io` would be better? (it redirects, I know)

Reply:
@migueldiascosta 

wow, good job with finding the url dup!

> url

done too.